lib/alerts_viewer_web/live/bus_live.html.heex

```diff
@@ -59,14 +59,18 @@
   <:col :let={route} label="Name">
     <%= Route.name(route) %>
   </:col>
-  <:col :let={route} label="Individual Vehicle Schedule Adherence (seconds)">
-    <%= @stats_by_route |> RouteStats.vehicles_schedule_adherence_secs(route) |> Enum.join(", ") %>
+  <:col :let={route} label="Individual Vehicle Schedule Adherence (minutes)">
+    <%= @stats_by_route
+    |> RouteStats.vehicles_schedule_adherence_secs(route)
+    |> Enum.map(&seconds_to_minutes(&1))
+    |> Enum.join(", ") %>
   </:col>
-  <:col :let={route} label="Median Schedule Adherence">
-    <%= RouteStats.median_schedule_adherence(@stats_by_route, route) %>
+  <:col :let={route} label="Median Schedule Adherence (minutes)">
+    <%= RouteStats.median_schedule_adherence(@stats_by_route, route) |> seconds_to_minutes %>
```

Comment:
```suggestion
    <%= RouteStats.median_schedule_adherence(@stats_by_route, route) |> seconds_to_minutes() %>
```

nitpick: I'm surprised the formatter doesn't complain about this, but I think the general preference is to include parens when calling a function like this.